.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "restructuredtext.confPath": "${workspaceFolder}/docs/venv/lib/python3.7/site-packages/pbr/tests/testpackage/doc/source"
```

Comment:
In the future probably worth adding editor files to .gitignore

Reply:
Oh, absolutely. I did not realize this was commited.